tests/lib/rook/upstream.py

```diff
@@ -69,9 +65,16 @@ def build(self):
                 % os.path.join(self.build_dir, 'rook-ceph.tar.gz'))
         self._rook_built = True
 
-    def upload_rook_image(self):
-        self.kubernetes.hardware.ansible_run_playbook(
-            "playbook_rook_upstream.yaml")
+    def preinstall(self):
+        super().preinstall()
+        if converter('@bool', settings.UPSTREAM_ROOK.BUILD_ROOK_FROM_GIT):
+            self.upload_rook_image()
+            self._fix_yaml()
+
+    def _get_charts(self):
+        name = f"rook-{settings.UPSTREAM_ROOK.HELM.CHANNEL}"
```

Comment:
CI is failing because HELM.CHANNEL is never set.

Reply:
This is a bad rebase left over. Sorry about that.